src/controllers/home.controller.js

```diff
@@ -35,7 +37,36 @@ const getCities = catchAsync(async (req, res) => {
   res.send(cities);
 });
 
+const paymentConfig = catchAsync(async (req, res) => {
+  res.send({
+    publishableKey: config.stripe.publishableKey,
+  });
+});
+
+const createPaymentIntent = catchAsync(async (req, res) => {
+  const { amount, currency } = req.body;
+  try {
+    const paymentIntent = await stripe.paymentIntents.create({
+      currency: 'EUR',
+      amount,
```

Comment:
This currency should not be hardcoded. 
And its value should be either `AED` or `INR`